src/views/Students/StudentComponents/AlumniServiceForm.jsx

```diff
@@ -230,6 +260,81 @@ const AlumniServiceForm = (props) => {
                       required={feeFieldsRequired}
                     />
                   </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="program_mode"
+                      label="Program Mode"
+                      placeholder="Program Mode"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={programOptions}
+                    />
+                  </div>
+                  <div className="col-md-6 col-sm-12 mt-2">
+                    <Input
+                      name="category"
+                      label="Category"
+                      placeholder="category"
+                      control="lookup"
+                      icon="down"
+                      className="form-control"
+                      options={categoryOptions}
+                      onChange={handleCategoryChange}
+                    />
+                  </div>
+                  {selectedCategory === "Youth Leadership" && (
+                    <div className="col-md-6 col-sm-12 mt-2">
+                      <Input
+                      icon="down"
+                      control="lookup"
+                      name="youth_leadership"
+                      label="Youth Leadership"
+                      options={youthLeadershipOptions}
+                      className="form-control"
+                      placeholder="Youth Leadership"
+                      />
+                    </div>
```

Comment:
instead of repeating the whole HTML blocks, I think you should instead use a switch-case block to set name, label, placeholder and options variables. And then use these in a single HTML field.

Something like:
```js
let subCategoryName = ''
let subCategoryLabel = ''
let subCategoryOptions = []
switch (selectedCategory) {
  case 'Youth Leadership':
    subCategoryName = 'youth_leadership'
    subCategoryLabel = 'Youth Leadership'
    subCategoryOptions = youthLeadershipOptions
    break;

  case 'Youth Leadership':
    subCategoryName = 'ideation_club'
    subCategoryLabel = 'Ideation Club'
    subCategoryOptions = ideationClubOptions
    break;
  
  // and so on...
}

return (
  // some code above
  // some code above
  <div className="col-md-6 col-sm-12 mt-2">
    <Input
      name={subCategoryName}
      label={subCategoryLabel}
      placeholder={subCategoryLabel}
      control="lookup"
      icon="down"
      className="form-control"
      options={subCategoryOptions}
    />
  </div>
  // some code below
  // some code below
)
```